src/Core/System/Snippet/Filter/TermFilter.php

```diff
@@ -27,11 +27,11 @@ public function filter(array $snippets, $requestFilterValue): array
         $result = [];
         foreach ($snippets as $setId => $set) {
             foreach ($set['snippets'] as $translationKey => $snippet) {
-                $term = sprintf('*%s*', $requestFilterValue);
```

Comment:
Can you add a Unit Test to cover this case?

Reply:
Will do!

Reply:
- [ ] TODO: Add Unit Test